Yum <command> fails in docker container

```
➜  app-service git:(master) make docker-image                                                                                                                                       [223/1097]
Sending build context to Docker daemon  259.6kB                                                
Step 1/27 : FROM centos:7 as builder                                                           
 ---> 9f38484d220f                                                                             
Step 2/27 : LABEL maintainer "Devtools <devtools@redhat.com>"                                  
 ---> Using cache                                                                              
 ---> 927f529e34e6                                                                             
Step 3/27 : LABEL author "Konrad Kleine <kkleine@redhat.com>"                                  
 ---> Using cache                                                                              
 ---> fb6f3c23eb69                                                                             
Step 4/27 : ENV LANG=en_US.utf8                                                                
 ---> Using cache                                                                              
 ---> b4cfd9fc6716                                                                             
Step 5/27 : ENV GOPATH /tmp/go                                                                 
 ---> Using cache                                                                              
 ---> 7305894f243b                                                                             
Step 6/27 : ARG GO_PACKAGE_PATH=github.com/redhat-developer/app-service                        
 ---> Using cache                                                                              
 ---> cdbf062fb062                                                                             
Step 7/27 : ARG VERBOSE=1                                                                      
 ---> Using cache                                                                                                                                                                             
 ---> 8fdcc272c002                                                                                                                                                                            
Step 8/27 : RUN yum install epel-release -y   && yum install --enablerepo=centosplus install -y --quiet     findutils     git     golang     make     procps-ng     tar     wget     which    
 bc   && yum clean all   && mkdir -p $GOPATH/bin && chmod a+rwx $GOPATH   && curl -L -s https://github.com/golang/dep/releases/download/v0.5.1/dep-linux-amd64 -o dep   && echo "7479cca72da0$
96bb3c23094d363ea32b7336daa5473fa785a2099be28ecd0e3  dep" > dep-linux-amd64.sha256   && sha256sum -c dep-linux-amd64.sha256   && rm dep-linux-amd64.sha256   && chmod +x ./dep   && mv dep $G$
PATH/bin/dep                                                                                   
 ---> Running in 1cb2a8e22517
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors


 One of the configured repositories failed (Unknown),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:

     1. Contact the upstream for the repository and get them to fix the problem.

     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer
        distribution release than is supported by the repository (and the
        packages for the previous distribution release still work).

     3. Run the command with the repository temporarily disabled
            yum --disablerepo=<repoid> ...

     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:

            yum-config-manager --disable <repoid>
        or                                     
            subscription-manager repos --disable=<repoid>

     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:                            

            yum-config-manager --save --setopt=<repoid>.skip_if_unavailable=true

Cannot find a valid baseurl for repo: base/7/x86_64
Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=container error was
12: Timeout on http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=container: (28, 'Resolving timed out after 30545 milliseconds')
The command '/bin/sh -c yum install epel-release -y   && yum install --enablerepo=centosplus install -y --quiet     findutils     git     golang     make     procps-ng     tar     wget     w
hich     bc   && yum clean all   && mkdir -p $GOPATH/bin && chmod a+rwx $GOPATH   && curl -L -s https://github.com/golang/dep/releases/download/v0.5.1/dep-linux-amd64 -o dep   && echo "7479c
ca72da0596bb3c23094d363ea32b7336daa5473fa785a2099be28ecd0e3  dep" > dep-linux-amd64.sha256   && sha256sum -c dep-linux-amd64.sha256   && rm dep-linux-amd64.sha256   && chmod +x ./dep   && mv
 dep $GOPATH/bin/dep' returned a non-zero code: 1
make: *** [Makefile:77: docker-image] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -75,6 +75,7 @@ test-coverage-html: ./vendor ./out/cover.out
 .PHONY: docker-image
 docker-image: Dockerfile
 	$(Q)docker build ${Q_FLAG} \
+		--network host \
 		--build-arg GO_PACKAGE_PATH=${GO_PACKAGE_PATH} \
 		--build-arg VERBOSE=${VERBOSE} \
 		. \
@@ -82,7 +83,7 @@ docker-image: Dockerfile
 
 .PHONY: docker-run
 docker-run: docker-image
-	$(Q)docker run -it --rm -p 8080:8080 ${GO_PACKAGE_ORG_NAME}/${GO_PACKAGE_REPO_NAME}:${GIT_COMMIT_ID}
+	$(Q)docker run -it --rm -p 8080:8080 --network host ${GO_PACKAGE_ORG_NAME}/${GO_PACKAGE_REPO_NAME}:${GIT_COMMIT_ID}
 
 .PHONY: docker-test
 docker-test: docker-image

